prow/plugins/lgtm/lgtm.go

```diff
@@ -115,23 +115,20 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 // optionsForRepo gets the plugins.Lgtm struct that is applicable to the indicated repo.
 func optionsForRepo(config *plugins.Configuration, org, repo string) *plugins.Lgtm {
 	fullName := fmt.Sprintf("%s/%s", org, repo)
-	for i := range config.Lgtm {
-		if !strInSlice(org, config.Lgtm[i].Repos) && !strInSlice(fullName, config.Lgtm[i].Repos) {
+	for _, c := range config.Lgtm {
+		if !sets.NewString(c.Repos...).Has(fullName) {
```

Comment:
Make the set once, not on every loop. Might also short-circuit with `!HasAny()`

Reply:
Thanks a lot for spotting this--it definitely seems more efficient and clean. I will correct this in another tiny PR. I really really appreciate your feedback